_monitoring-your-cluster/pa/reference.md

```diff
@@ -651,6 +651,140 @@ This list is extensive. We recommend using Ctrl/Cmd + F to find what you're look
       <td>FollowerCheck_Latency (WIP)
       </td>
     </tr>
+    <tr>
+      <td>searchbp_shard_stats_cancellationCount
+      </td>
+      <td rowspan="22">NodeID, searchbp_mode
+      </td>
+      <td>The number of tasks marked for cancellation, on search shard task level.
+      </td>
+    </tr>
+    <tr>
+      <td>searchbp_shard_stats_limitReachedCount
+      </td>
+      <td>The number of times when the number of tasks eligible for cancellation exceeded the set cancellation threshold, on search shard task level.
```

Comment:
The number of times when **cancellable task total(sum?)** exceeded the set cancellation threshold, on search shard task level.